core/chainlink.Dockerfile

```
@@ -1,5 +1,5 @@
 # Build image: Chainlink binary
-FROM golang:1.21-bullseye as buildgo
+FROM golang:1.21.7-bullseye as buildgo
```

`1.21` is preferred because it is always the latest patch version:
```suggestion
FROM golang:1.21-bullseye as buildgo
```